packages/ddp-client/client/client_convenience.js

```diff
@@ -42,6 +43,9 @@ Meteor.connection = DDP.connect(ddpUrl, {
   onDDPVersionNegotiationFailure: onDDPVersionNegotiationFailure
 });
 
+// Makes sure Meteor.connect is already created before injecting the helpers
```

Comment:
If the plan is to wrap the `Connection` methods instead of integrating the queue into the `Connection` class, it would probably be better to instead wrap the methods on Connection.prototype and do it before creating `Meteor.connection`. This would ensure that it works the same way for every DDP client, including `Meteor.connection`.

Reply:
That sounds good. I'll implement it.